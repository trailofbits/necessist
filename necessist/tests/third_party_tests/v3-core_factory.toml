url = "https://github.com/Uniswap/v3-core"
test_files = ["test/UniswapV3Factory.spec.ts"]
full = true
