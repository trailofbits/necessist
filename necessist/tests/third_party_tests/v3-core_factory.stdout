43 candidates in 1 test file
$DIR/test/UniswapV3Factory.spec.ts: dry running
$DIR/test/UniswapV3Factory.spec.ts: mutilating
$DIR/test/UniswapV3Factory.spec.ts:44:25-44:33: `.owner()` failed
$DIR/test/UniswapV3Factory.spec.ts:48:26-48:60: `.provider.getCode(factory.address)` failed
$DIR/test/UniswapV3Factory.spec.ts:52:5-52:85: `await factory.createPool(TEST_ADDRESSES[0], TEST_ADDRESSES[1], FeeAmount.MEDIUM)` failed
$DIR/test/UniswapV3Factory.spec.ts:52:18-52:85: `.createPool(TEST_ADDRESSES[0], TEST_ADDRESSES[1], FeeAmount.MEDIUM)` failed
$DIR/test/UniswapV3Factory.spec.ts:54:26-54:56: `.provider.getCode(poolAddress)` failed
$DIR/test/UniswapV3Factory.spec.ts:58:25-58:61: `.feeAmountTickSpacing(FeeAmount.LOW)` failed
$DIR/test/UniswapV3Factory.spec.ts:59:25-59:64: `.feeAmountTickSpacing(FeeAmount.MEDIUM)` failed
$DIR/test/UniswapV3Factory.spec.ts:60:25-60:62: `.feeAmountTickSpacing(FeeAmount.HIGH)` failed
$DIR/test/UniswapV3Factory.spec.ts:91:7-91:62: `await createAndCheckPool(TEST_ADDRESSES, FeeAmount.LOW)` passed
$DIR/test/UniswapV3Factory.spec.ts:95:7-95:65: `await createAndCheckPool(TEST_ADDRESSES, FeeAmount.MEDIUM)` passed
$DIR/test/UniswapV3Factory.spec.ts:98:7-98:63: `await createAndCheckPool(TEST_ADDRESSES, FeeAmount.HIGH)` passed
$DIR/test/UniswapV3Factory.spec.ts:102:7-102:89: `await createAndCheckPool([TEST_ADDRESSES[1], TEST_ADDRESSES[0]], FeeAmount.MEDIUM)` passed
$DIR/test/UniswapV3Factory.spec.ts:106:27-106:91: `.createPool(TEST_ADDRESSES[0], TEST_ADDRESSES[0], FeeAmount.LOW)` failed
$DIR/test/UniswapV3Factory.spec.ts:110:27-110:95: `.createPool(TEST_ADDRESSES[0], constants.AddressZero, FeeAmount.LOW)` failed
$DIR/test/UniswapV3Factory.spec.ts:111:27-111:95: `.createPool(constants.AddressZero, TEST_ADDRESSES[0], FeeAmount.LOW)` failed
$DIR/test/UniswapV3Factory.spec.ts:112:27-112:99: `.createPool(constants.AddressZero, constants.AddressZero, FeeAmount.LOW)` failed
$DIR/test/UniswapV3Factory.spec.ts:118:27-118:81: `.createPool(TEST_ADDRESSES[0], TEST_ADDRESSES[1], 250)` failed
$DIR/test/UniswapV3Factory.spec.ts:122:7-122:104: `await snapshotGasCost(factory.createPool(TEST_ADDRESSES[0], TEST_ADDRESSES[1], FeeAmount.MEDIUM))` passed
$DIR/test/UniswapV3Factory.spec.ts:122:36-122:103: `.createPool(TEST_ADDRESSES[0], TEST_ADDRESSES[1], FeeAmount.MEDIUM)` failed
$DIR/test/UniswapV3Factory.spec.ts:128:27-128:42: `.connect(other)` failed
$DIR/test/UniswapV3Factory.spec.ts:128:42-128:67: `.setOwner(wallet.address)` failed
$DIR/test/UniswapV3Factory.spec.ts:132:7-132:44: `await factory.setOwner(other.address)` failed
$DIR/test/UniswapV3Factory.spec.ts:132:20-132:44: `.setOwner(other.address)` failed
$DIR/test/UniswapV3Factory.spec.ts:133:27-133:35: `.owner()` failed
$DIR/test/UniswapV3Factory.spec.ts:137:27-137:51: `.setOwner(other.address)` failed
$DIR/test/UniswapV3Factory.spec.ts:143:7-143:44: `await factory.setOwner(other.address)` failed
$DIR/test/UniswapV3Factory.spec.ts:143:20-143:44: `.setOwner(other.address)` failed
$DIR/test/UniswapV3Factory.spec.ts:144:27-144:52: `.setOwner(wallet.address)` failed
$DIR/test/UniswapV3Factory.spec.ts:150:27-150:42: `.connect(other)` failed
$DIR/test/UniswapV3Factory.spec.ts:150:42-150:66: `.enableFeeAmount(100, 2)` failed
$DIR/test/UniswapV3Factory.spec.ts:153:27-153:56: `.enableFeeAmount(1000000, 10)` failed
$DIR/test/UniswapV3Factory.spec.ts:156:27-156:51: `.enableFeeAmount(500, 0)` failed
$DIR/test/UniswapV3Factory.spec.ts:159:27-159:55: `.enableFeeAmount(500, 16834)` failed
$DIR/test/UniswapV3Factory.spec.ts:162:7-162:44: `await factory.enableFeeAmount(100, 5)` failed
$DIR/test/UniswapV3Factory.spec.ts:162:20-162:44: `.enableFeeAmount(100, 5)` failed
$DIR/test/UniswapV3Factory.spec.ts:163:27-163:52: `.enableFeeAmount(100, 10)` failed
$DIR/test/UniswapV3Factory.spec.ts:166:7-166:44: `await factory.enableFeeAmount(100, 5)` failed
$DIR/test/UniswapV3Factory.spec.ts:166:20-166:44: `.enableFeeAmount(100, 5)` failed
$DIR/test/UniswapV3Factory.spec.ts:167:27-167:53: `.feeAmountTickSpacing(100)` failed
$DIR/test/UniswapV3Factory.spec.ts:170:27-170:51: `.enableFeeAmount(100, 5)` failed
$DIR/test/UniswapV3Factory.spec.ts:173:7-173:45: `await factory.enableFeeAmount(250, 15)` failed
$DIR/test/UniswapV3Factory.spec.ts:173:20-173:45: `.enableFeeAmount(250, 15)` failed
$DIR/test/UniswapV3Factory.spec.ts:174:7-174:80: `await createAndCheckPool([TEST_ADDRESSES[0], TEST_ADDRESSES[1]], 250, 15)` passed
